airgap/api.go

```diff
@@ -136,13 +137,39 @@ func (tx *Transaction) AsGethTransaction() *types.Transaction {
 		tx.GatewayFee,
 		tx.Data,
 	)
+	if tx.Signed() {
+		signer := types.NewEIP155Signer(tx.ChainId)
+		signedGethTx, err := gethTx.WithSignature(signer, tx.GetSigBytes())
+		if err != nil {
+			return nil, err
+		}
+		return signedGethTx, nil
+	}
+	return gethTx, nil
 }
 
-func (tx *Transaction) Hash() common.Hash {
-	return tx.AsGethTransaction().Hash()
+func (tx *Transaction) GetSigBytes() []byte {
```

Comment:
i think it's simpler to change `Transaction` type to store `signature []byte` instead of `V,R,S` and then have a `GetSignatureValue() (v,r,s)` which is just calling ehtereum transaction